src/ml_tooling/baseclass.py

```diff
@@ -94,7 +117,7 @@ def load_estimator(cls, path: str) -> "Model":
         logger.info(f"Loaded {instance.estimator_name}")
         return instance
 
-    def save_estimator(self, path: str) -> pathlib.Path:
+    def save_estimator(self, path: Union[str, pathlib.Path]) -> pathlib.Path:
```

Comment:
This is not reflected in the docstring.

Reply:
What happens if file already exists? Maybe we should raise?

Reply:
Not touching the save_estimator and load_estimator yet, as that is handled by #292